client/me/purchases/billing-history/receipt.jsx

```diff
@@ -318,7 +318,7 @@ function ReceiptLineItems( { transaction } ) {
 				<tfoot>
 					<tr>
 						<td className="billing-history__receipt-desc">
-							<strong>{ translate( 'Total' ) }:</strong>
+							<strong>{ translate( 'Total Paid' ) }:</strong>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **30** times:
`translate( 'Total paid', { context: 'Sum of earnings that have been distributed'} )` *ES Score: 11*